tests/Utilities.Test.UnitTests/GenericFactoryTests.cs

```diff
@@ -87,11 +87,16 @@ public void RegisterOrReplaceService_ServiceNull_ArgumentNullException()
             // Arrange
             GenericFactory factory = new GenericFactory();
 
-            ITestDependencyInterface? dependency = null;
+
 
             // Act
+#if NET462 || NET48 || NETSTANDARD20 || NETSTANDARD21
+            ITestDependencyInterface dependency = null;
```

Comment:
Moet deze niet ook nullable?

Reply:
kan pas vanaf C# 8 wat niet in framework zit.
Daarnaast sidenote; heb deze in de zelfde test file gedaan zodat de tests wel door dezelfde tests gechecked worden. Anders heb je per framework versie potentieel een test file. Dat vind ik minder.